File: 백준/Bronze/10810. 공 넣기/공 넣기.py
import sys
input = sys.stdin.readline

n,m = map(int, input().split())
ball = []
pocket = [0] * n

for i in range(m):
    ball.append(list(map(int, input().split())))
    
for i in range(m):
    start, end = ball[i][0], ball[i][1]
    for j in range(start-1, end):
        pocket[j] = ball[i][2] 
        
print(*pocket[0:], sep=" ")

File: 백준/Bronze/10810. 공 넣기/공 넣기.swift
let input = readLine()!.split(separator: " ").map { Int($0)! }
let (N,M) = (input[0],input[1])

var basket = Array(repeating:0, count: N)
var ball = [[Int]]()
for _ in 0..<M {
    ball.append(readLine()!.split(separator: " ").map { Int($0)! })
}

for i in ball {
    for j in i[0]...i[1] {
        basket[j-1] = i[2]
    }
}
print(basket.map { String($0) }.joined(separator: " "))